llrt_core/src/vm.rs

```diff
@@ -184,6 +184,28 @@ impl Vm {
 
         self.run(source, strict, global).await;
     }
+
+    pub async fn run_repl(&self) {
+        self.ctx
+            .with(|ctx| -> Result<()> {
+                let global = ctx.globals();
+                let console: Object = global.get("console")?;
+                let js_log: Function = console.get("log")?;
```

Comment:
We can call these methods directly from rust and print. Check the console module.